src/tidy.zig

```diff
@@ -600,24 +600,37 @@ test "tidy extensions" {
     });
 
     const exceptions = std.StaticStringMap(void).initComptime(.{
-        .{".editorconfig"},                       .{".gitignore"},
-        .{".nojekyll"},                           .{"CNAME"},
-        .{"Dockerfile"},                          .{"exclude-pmd.properties"},
-        .{"favicon.ico"},                         .{"favicon.png"},
-        .{"LICENSE"},                             .{"module-info.test"},
-        .{"index.html"},                          .{"logo.svg"},
-        .{"logo-white.svg"},                      .{"logo-with-text-white.svg"},
-        .{"zig/download.sh"},                     .{"src/scripts/cfo_supervisor.sh"},
-        .{"src/docs_website/scripts/build.sh"},   .{".github/ci/docs_check.sh"},
-        .{".github/ci/test_aof.sh"},              .{"tools/systemd/tigerbeetle-pre-start.sh"},
-        .{"tools/vscode/format_debug_server.sh"},
+        .{".editorconfig"},                 .{".gitignore"},
+        .{".nojekyll"},                     .{"CNAME"},
+        .{"exclude-pmd.properties"},        .{"favicon.ico"},
+        .{"favicon.png"},                   .{"LICENSE"},
+        .{"module-info.test"},              .{"index.html"},
+        .{"logo.svg"},                      .{"logo-white.svg"},
+        .{"logo-with-text-white.svg"},      .{"zig/download.sh"},
+        .{"src/scripts/cfo_supervisor.sh"}, .{"src/docs_website/scripts/build.sh"},
+        .{".github/ci/docs_check.sh"},      .{".github/ci/test_aof.sh"},
     });
 
     const allocator = std.testing.allocator;
     const shell = try Shell.create(allocator);
     defer shell.destroy();
 
     const paths = try list_file_paths(shell);
+
+    for (exceptions.keys()) |exception| {
+        for (paths) |path| {
```

Comment:
The `O(mn)` can serve as motivation to keep our exception list below a few million :smile:.